scripts/check_frontend_coverage.py

```diff
@@ -537,35 +590,38 @@ def check_coverage_changes():
                         ' file does not exist.'.format(LCOV_FILE_PATH))
 
     stanzas = get_stanzas_from_lcov_file()
-    whitelist = list(FULLY_COVERED_FILENAMES)
+    blacklist = list(NOT_FULLY_COVERED_FILENAMES)
     errors = ''
 
     for stanza in stanzas:
         file_name = stanza.file_name
         total_lines = stanza.total_lines
         covered_lines = stanza.covered_lines
 
-        if file_name in whitelist:
+        if file_name not in blacklist:
             if total_lines != covered_lines:
-                errors += ('\033[1m{}\033[0m file is in the whitelist but its'
-                           ' coverage decreased. Make sure it is fully covered'
-                           ' by Karma unit tests.\n'.format(file_name))
-
-            whitelist.remove(file_name)
+                errors += ('\033[1m{}\033[0m seems to be not completely tested.'
+                           ' Make sure it\'s fully covered before pushing'
+                           ' your changes.\n'.format(file_name))
         else:
             if total_lines == covered_lines:
-                errors += ('\033[1m{}\033[0m file is fully covered but it\'s'
-                           ' not in the "100% coverage" whitelist. Please add'
-                           ' the file name in the whitelist in the file'
-                           ' scripts/check_frontend_test_coverage.py.\n'
+                errors += ('\033[1m{}\033[0m seems to be fully covered!'
+                           ' Before removing it manually from the blacklist'
+                           ' in the file'
+                           ' scripts/check_frontend_test_coverage.py, please'
+                           ' make sure you\'ve followed the unit tests rules'
+                           ' correctly on:'
+                           ' https://github.com/oppia/oppia/wiki/Frontend-unit-tests-guide#rules\n' # pylint: disable=line-too-long
                            .format(file_name))
 
-    if whitelist:
-        for test_name in whitelist:
+            blacklist.remove(file_name)
+
+    if blacklist:
+        for test_name in blacklist:
             errors += ('\033[1m{}\033[0m is in the frontend test coverage'
-                       ' whitelist but it doesn\'t exist anymore. If you have'
+                       ' blacklist but it doesn\'t exist anymore. If you have'
                        ' renamed it, please make sure to remove the old file'
-                       ' name and add the new file name in the whitelist in'
+                       ' name and add the new file name in the blacklist in'
                        ' the file scripts/check_frontend_test_coverage.py.\n'
                        .format(test_name))
 
```

Comment:
Update the docstring for the main function accordingly.

Reply:
Done.

Reply:
Yes, it's resolved. Actually I don't remember the reason of this comment, can you please help me out here?@seanlip

Reply:
I tried `git blame` ([link](https://github.com/oppia/oppia/pull/8130/files#diff-c213973e0689047b1c20b4448b6f75d6R474)) and found that you have written this line.

I think that's a mistake and it should be `check_frontend_coverage.py` instead of `wrap_up_release.py`.

Reply:
Done.